contentcuration/contentcuration/frontend/channelEdit/components/edit/DetailsTabView.vue

```diff
@@ -734,6 +741,17 @@
       isUnique(value) {
         return value !== nonUniqueValue;
       },
+      allowedFileType(file) {
+        let allowedFileTypes = [];
+        // add the relevant format presets for audio and video
+        // high res and low res are currently the same, so only one is included
+        allowedFileTypes.push(
+          FormatPresetsMap.get(FormatPresetsNames.HIGH_RES_VIDEO).allowed_formats
+        );
+        allowedFileTypes.push(FormatPresetsMap.get(FormatPresetsNames.AUDIO).allowed_formats);
+        allowedFileTypes = allowedFileTypes.flat();
```

Comment:
TIL `.flat()` exists on `Array.prototype`

Reply:
A slight optimization would be to only create this `allowedFileTypes` array once, outside of this function scope, but I think it's unlikely to have major performance differences